rust/feature-flags/src/cohort_operations.rs

```diff
@@ -0,0 +1,369 @@
+use std::collections::HashSet;
+use std::sync::Arc;
+use tracing::instrument;
+
+use crate::cohort_models::{Cohort, CohortId, CohortProperty, InnerCohortProperty};
+use crate::{api::FlagError, database::Client as DatabaseClient, flag_definitions::PropertyFilter};
+
+impl Cohort {
+    /// Returns a cohort from postgres given a cohort_id and team_id
+    #[instrument(skip_all)]
+    pub async fn from_pg(
```

Comment:
I'd slap a `#[cfg(test)]` on this - you never want to load cohorts individually in production, and you don't currently anywhere, but that lets you make sure nobody ever will.

Alternatively I'd delete it altogether.